pkg/queue/manager.go

```diff
@@ -109,6 +123,12 @@ func (m *Manager) UpdateClusterQueue(cq *kueue.ClusterQueue) error {
 	}
 	// TODO(#8): recreate heap based on a change of queueing policy.
 	cqImpl.Update(cq)
```

Comment:
what if this changes the cohort?

Reply:
I add the part to update the cohorts in manager.
But I'm curious if we need to deny users from updating the cohort of ClusterQueue in the future.